### DISCUSSION TITLE
Question sur les conflits entre logiciels

### DISCUSSION THREAD

**Discussion by EdenTrismegiste** [marian-m12l/studio#495] [Q&A]
Bonjour,

J'ai fait l'acquisition d'une Lunii auprès d'un particulier qui a laissé installé dessus des packs d'histoires.
Ces packs d'histoires sont des packs officiels, et après lecture des différents sujets ici, je comprends que leur extraction ne se fait que par le logiciel officiel. Pouvez vous confirmer ce premier point?

Le fait de connecter la Lunii a mon compte supprimera-t-il les histoires préalablement installées sur la Lunii? Est-il possible de les récupérer pour les utiliser à posteriori?
Est-il possible de faire cohabité des histoires non officielles (transférées par Studio) et des histoires officielles?

En vous remerciant d'avance pour votre aide !
Cordialement,

**Comment by marian-m12l** [comment]
Bonjour,

Effectivement, l'idée est de passer par le logiciel officiel pour gérer vos packs officiels. De ce que je comprends, les histoires présentes sur une fabrique à histoires d'occasion ne sont pas transférées vers le compte du nouveau propriétaire.

En ce qui concerna la cohabitation entre histoires officielles et non-officielles, cela ne pose pas de problème. En revanche, Il est possible que le modèle wifi supprime les histoires non reconnues (#460).
